CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ ENDFOREACH()
 ADD_SUBDIRECTORY(deps/googletest)
 INCLUDE_DIRECTORIES(${gtest_SOURCE_DIR}/include/ ${gtest_SOURCE_DIR})
 
+# 3rd party modules
+ADD_SUBDIRECTORY(deps/aslam_cameras)
```

Comment:
For now I want importing them to be the same as if kalibr was installed separately. Including it in `deps` is just a convenience.